### PR TITLE
OCPQE-24595 add check for empty release list

### PR DESCRIPTION
**Pull request by tomasdavidorg**
Fix error when release list is empty.

[OCPQE-24595](https://issues.redhat.com/browse/OCPQE-24595)

### CODE REVIEW

File: oar/controller/detector.py [hunk 1]
```diff
@@ -29,7 +29,11 @@ def get_latest_zstream_version(self):
         resp = requests.get(url)
         if resp.ok:
             yamlobj = yaml.safe_load(resp.text)
-            return list(yamlobj["releases"].keys())[-1]
+            releases = list(yamlobj["releases"].keys())
+            if len(releases) > 1:
```
Comment (rioliu-rh) [inline oar/controller/detector.py:33]:
```suggestion
            if len(releases):
```

**Comment by rioliu-rh** [comment]
4.17 is latest release version, we should not run detector for this release.

**Comment by tomasdavidorg** [comment]
> 4.17 is latest release version, we should not run detector for this release.

Yes, I just want to cover the case when someone tries to use the tool against the unreleased version which already has [the yaml file ](https://github.com/openshift/release-tests/blob/z-stream/_releases/4.17/4.17.z.yaml) created.

**Pull request merged**